### ISSUE TITLE
Container crashes

### ISSUE THREAD

**Issue by tobiasff3200** [RouHim/this-week-in-past#58]
Hi
I have startet the docker container with the `docker-compose.yml` you provide in the Readme.
The container starts fine but crashed if I try to access it with a browser.
The error is:
```
tobias@server:~/this-week-in-past$ docker compose up
[+] Running 1/0
 ⠿ Container this-week-in-past-this-week-in-past-1  Created                                                                                                                                                                              0.0s
Attaching to this-week-in-past-this-week-in-past-1
this-week-in-past-this-week-in-past-1  | Begin fetch_images
this-week-in-past-this-week-in-past-1  | Purging kv store
this-week-in-past-this-week-in-past-1  | Indexing resources, this may take some time depending on the amount of resources...
this-week-in-past-this-week-in-past-1  | Storing 24964 items
this-week-in-past-this-week-in-past-1  | Cleanup cache
this-week-in-past-this-week-in-past-1  | Job done in 56s!
this-week-in-past-this-week-in-past-1  | Launching webserver 🚀
this-week-in-past-this-week-in-past-1  | thread 'actix-rt|system:0|arbiter:1' panicked at 'called `Result::unwrap()` on an `Err` value: Error("invalid type: null, expected f32", line: 1, column: 331)', src/resource_processor.rs:19:84
this-week-in-past-this-week-in-past-1  | note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
If I access it with `curl localhost:8080` it works.

**Comment by tobiasff3200** [comment]
Thank you for the fast reply. Maybe the application could just skip the corrupted image.

**Comment by tobiasff3200** [comment]
That's perfect. With 24964 images, the search could take a while. :sweat_smile:

**Comment by RouHim** [comment]
Just for interest, how long does the indexing take?

**Comment by RouHim** [comment]
Awesome thanks

**Comment by RouHim** [comment]
Will be fixed within feature #48

**Comment by RouHim** [comment]
@tobiasff3200 are you able to reproduce the bug? If so you can try the next gen container image: 
`rouhim/this-week-in-past:next`

**Comment by tobiasff3200** [comment]
I get a different error now:
```
tobias@server:~/this-week-in-past$ docker compose up
[+] Running 1/0
 ⠿ Container this-week-in-past-this-week-in-past-1  Created                                                                                                                                                                              0.0s
Attaching to this-week-in-past-this-week-in-past-1
this-week-in-past-this-week-in-past-1  | [2022-11-10T19:42:39Z INFO  this_week_in_past] Launching webserver 🚀
this-week-in-past-this-week-in-past-1  | [2022-11-10T19:42:39Z INFO  this_week_in_past::scheduler] Begin resource indexing
this-week-in-past-this-week-in-past-1  | [2022-11-10T19:42:39Z INFO  this_week_in_past::scheduler] Indexing resources, this may take some time depending on the amount of resources...
this-week-in-past-this-week-in-past-1  | [2022-11-10T19:42:39Z INFO  actix_server::builder] Starting 4 workers
this-week-in-past-this-week-in-past-1  | [2022-11-10T19:42:39Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
this-week-in-past-this-week-in-past-1  | [2022-11-10T19:43:25Z INFO  this_week_in_past::scheduler] Found 24963 resources
this-week-in-past-this-week-in-past-1  | [2022-11-10T19:43:25Z INFO  this_week_in_past::scheduler] Purging resources store
this-week-in-past-this-week-in-past-1  | [2022-11-10T19:43:25Z INFO  this_week_in_past::scheduler] Cleanup cache
this-week-in-past-this-week-in-past-1  | [2022-11-10T19:43:25Z INFO  this_week_in_past::scheduler] Inserting new resources
this-week-in-past-this-week-in-past-1  | [2022-11-10T19:43:26Z INFO  this_week_in_past::scheduler] Job done in 46s!
this-week-in-past-this-week-in-past-1  | [2022-11-10T19:43:33Z INFO  actix_web::middleware::logger] XXX.XXX.XXX.XXX "GET / HTTP/1.1" 304 0 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:106.0) Gecko/20100101 Firefox/106.0" 0.001562
this-week-in-past-this-week-in-past-1  | [2022-11-10T19:43:34Z INFO  actix_web::middleware::logger] XXX.XXX.XXX.XXX "GET /api/weather HTTP/1.1" 200 4 "http://XXX.XXX.XXX.XXX:8080/" "Mozilla/5.0 (X11; Linux x86_64; rv:106.0) Gecko/20100101 Firefox/106.0" 0.000262
this-week-in-past-this-week-in-past-1  | [2022-11-10T19:43:34Z INFO  actix_web::middleware::logger] XXX.XXX.XXX.XXX "GET /api/config/interval/refresh HTTP/1.1" 200 3 "http://XXX.XXX.XXX.XXX:8080/" "Mozilla/5.0 (X11; Linux x86_64; rv:106.0) Gecko/20100101 Firefox/106.0" 0.000055
this-week-in-past-this-week-in-past-1  | thread 'actix-rt|system:0|arbiter:0' panicked at 'called `Result::unwrap()` on an `Err` value: SqliteFailure(Error { code: SystemIoFailure, extended_code: 6410 }, Some("disk I/O error"))', src/resource_store.rs:35:43
this-week-in-past-this-week-in-past-1  | note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by tobiasff3200** [comment]
The new version also crashes if a http request comes in while the indexing is still in progress
```
thread 'actix-rt|system:0|arbiter:0' panicked at 'called `Option::unwrap()` on a `None` value', src/resource_endpoint.rs:71:10
```

**Comment by RouHim** [comment]
Great thank you, that helped me a lot <3

**Comment by RouHim** [comment]
@tobiasff3200 pushed a new release:
`rouhim/this-week-in-past:next`
Watch out it is still hot :D

**Comment by tobiasff3200** [comment]
Works perfect, thank you very much!

**Comment by RouHim** [comment]
Finally closed with #55
